### PR TITLE
fix refresh bug

### PR DESCRIPTION
**Pull request by jdneo**
1. fix refresh bug
2. add telemetry in redis cache node

#440

### CODE REVIEW

**Comment by voidfunction** [comment]
Can one of the admins verify this patch?

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/actions/CreateRedisCacheAction.java [hunk 1]
```diff
@@ -32,17 +32,26 @@
...
         try {
-            CreateRedisCacheForm dialog = new CreateRedisCacheForm(PluginUtil.getParentShell());
-            dialog.create();
-            dialog.open();
+            CreateRedisCacheForm createRedisCacheForm = new CreateRedisCacheForm(PluginUtil.getParentShell());
+            createRedisCacheForm.create();
+            createRedisCacheForm.setOnCreate(new Runnable() {
+                @Override
+                public void run() {
+                    redisCacheModule.load(false);
```
Comment (adashen) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/actions/CreateRedisCacheAction.java]: check null for redisCacheModule

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/serviceexplorer/azure/rediscache/CreateRedisCacheAction.java [hunk 2]
```diff
@@ -46,7 +45,7 @@ public void actionPerformed(NodeActionEvent e) {
         Project project = (Project) redisCacheModule.getProject();
         try {
             if (!AzureSignInAction.doSignIn(AuthMethodManager.getInstance(), project)) return;
-            CreateRedisCacheForm createRedisCacheForm = new CreateRedisCacheForm(project);
+            CreateRedisCacheForm createRedisCacheForm = new CreateRedisCacheForm((Project) redisCacheModule.getProject());
```
Comment (adashen) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/serviceexplorer/azure/rediscache/CreateRedisCacheAction.java]: null check?

**Comment by voidfunction** [comment]
Can one of the admins verify this patch?

**Pull request merged**